mycore-wcms2/src/main/java/org/mycore/wcms2/resources/MCRWCMSFileBrowserResource.java

```diff
@@ -124,7 +124,7 @@ public String getFolders() throws IOException, ParserConfigurationException, SAX
         DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
         File file = new File(MCRLayoutUtilities.getNavigationURL().getPath());
         Document doc = docBuilder.parse(file);
-        getAllowedPaths(doc.getDocumentElement());
+        retrieveAllowedPaths(doc.getDocumentElement());
```

Comment:
Better name would be: updateFolderList()

Reply:
done!